src/Components/Input.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react'
+import './Input.css'
+
+function Input({ setTodos }) {
+
+    const [newItem, setNewItem] = useState("")
+
+    const handleChange = (event) => {
+        setNewItem(event.target.value)
+    }
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        if (newItem !== "") {
+            setTodos((prevTodos) => [...prevTodos, {
```

Comment:
unnecessary array and spread as map already return and array.

Reply:
spread operator removed